packages/blade/src/components/TopNav/docs/TopNav.stories.tsx

```diff
@@ -220,128 +220,198 @@ const TopNavFullExample = () => {
-                <Box display="flex" gap="spacing.4" padding="spacing.4" alignItems="center">
-                  <Avatar size="medium" name="John Doe" />
-                  <Box display="flex" flexDirection="column" gap="spacing.2">
-                    <Text size="medium" weight="semibold">
-                      John Doe
-                    </Text>
-                    <Text size="xsmall" color="surface.text.gray.muted">
-                      Razorpay Trusted Merchant
-                    </Text>
-                  </Box>
-                </Box>
-                <MenuItem>
-                  <Text color="surface.text.gray.subtle">Settings</Text>
-                </MenuItem>
-                <MenuItem color="negative">
-                  <Text color="feedback.text.negative.intense">Logout</Text>
-                </MenuItem>
-              </MenuOverlay>
-            </Menu>
-          </TopNavActions>
+            </>
+          ) : (
+            <>
+              <TopNavBrand>
+                <RazorpayLogo />
+              </TopNavBrand>
+              <TopNavContent>
+                <TabNav
+                  items={[
+                    { title: 'Home', href: '/home', icon: HomeIcon },
+                    {
+                      href: '/payroll',
+                      title: 'Payroll',
+                      icon: RazorpayxPayrollIcon,
+                      description: 'Automate payroll with ease.',
+                    },
+                    {
+                      href: '/payments',
+                      title: 'Payments',
+                      icon: AcceptPaymentsIcon,
+                      description: 'Manage payments effortlessly.',
+                    },
+                    {
+                      href: '/magic-checkout',
+                      title: 'Magic Checkout',
+                      icon: MagicCheckoutIcon,
+                      description: 'Fast, one-click checkout.',
+                    },
+                    {
+                      href: '/rize',
+                      title: 'Rize',
+                      icon: AwardIcon,
+                      isAlwaysOverflowing: true,
+                      description: 'Boost your business growth.',
+                    },
+                  ]}
+                >
+                  {({ items, overflowingItems }) => {
+                    const activeProduct = overflowingItems.find(
+                      (item) => item.href === selectedProduct,
+                    );
+                    return (
+                      <>
+                        <TabNavItems>
+                          {items.map((item) => {
+                            return (
+                              <TabNavItemLink
+                                key={item.title}
+                                title={item.title}
+                                href={item.href}
+                                icon={item.icon}
+                              />
```

Comment:
No need to pass `isActive` here?

Reply:
Active state is managed by URL via `TabNavItemLink` component wrapper